conduct/code-of-conduct.md

```diff
@@ -1,17 +1,21 @@
 # Code of Conduct
```

Comment:
At the top, maybe include paragraph saying that this is a template and that you need to replace `<X>`, `<Y>`, ... with values for your event?